package.json

```diff
@@ -13,6 +13,17 @@
     "android": "react-native run-android",
     "snyk-protect": "snyk protect"
   },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
```

Comment:
- Instead of running tests manually, we should separate `"ci": "eslint . && jest && codecov"` into `eslint . && jest` and `codecov`.
For example:
```
// scripts
"pre-commit": "eslint . && jest",
"ci": "npm pre-commit && codecov",

// hooks
"pre-commit": "npm pre-commit"
```

- We should run it everytime, instead of `*.js` only.
For example: RN may add a11y that will break jest tests, because nothing has changed in js, but RN update changed view structure.